src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -132,15 +132,17 @@ the MIT License. See LICENSE in the project root for license information. -->
     </StackPanel.Resources>
 
     <Button Height="36.0" MinWidth="46.0" Width="46.0" x:Name="MinimizeButton" Style="{StaticResource CaptionButton}" Click="_MinimizeClick"
-            AutomationProperties.Name="Minimize">
+            AutomationProperties.Name="Minimize"

```

Comment:
This isn't localizable, but this is probably okay because it's Raw?

Reply:
I should probably localize it anyways, tbh. It can still be accessed.